### DISCUSSION TITLE
Remove content from fields

### DISCUSSION THREAD

**Discussion by Takaya213** [engram-design/FeedMe#872] [Ideas]
Is it possible to remove certain types of content from a field e.g. in the feed at the end of the description there is a Continue Reading link (from a WP site), is it possible to remove this from the imported entries dynamically?

Thanks

**Comment by engram-design** [comment]
Nothing native to take care of that at the moment, but you could do this yourself via a plugin. Refer to https://sgroup.com.au/plugins/feedme/developers/events-reference where you can use either `onStepProcessFeed` or `onProcessFeed` to trigger code after a single feed, or all feed items have run.

The alternative option is to parse your feed data beforehand, stripping out the Continue Reading link before it even gets to Feed Me. For example - your feed might be in JSON, you create a PHP script to read in the contents of your feed, you strip out any unneeded data, then output as JSON. Your PHP script would then become the URL for your feed to put into Feed Me.

**Comment by Takaya213** [comment]
Thanks for that, but my working knowledge of actual plugin development is kinda nothing as a front-end dev, I am sorry to say.
